### ISSUE TITLE
Referee 1.4

### ISSUE THREAD

**Issue by cgreene** [greenelab/deep-review#686]
> The authors mentioned “One source of training examples with rich clinical annotations is electronic health records”. What does it mean by “rich clinical annotations”? Can the authors provide a definition and a few examples.

**Comment by cgreene** [comment]
This should probably say _rich phenotypic information_ instead of clinical annotations. @brettbj I think you are the domain expert. What do you think?

**Comment by brettbj** [comment]
I'm having trouble with formatting for my pull request (w/ another one outstanding). Copying the text below in the event someone gets to it before I can resolve:

One source of training examples with rich phenotypical annotations is EHRs.
Billing information in the form of ICD codes are a simple annotation but
phenotypic algorithms can combine laboratory tests, medication prescriptions,
and patient notes to generate more reliable phenotypes. Recently, Lee et al.
[@tag:Lee2016_emr_oct_amd] developed an approach to distinguish individuals
with age-related macular degeneration from control individuals. They trained a
deep neural network on approximately 100,000 images extracted from structured
electronic health records, reaching greater than 93% accuracy. The authors used
their test set to evaluate when to stop training. In other domains, this has
resulted in a minimal change in the estimated accuracy
[@tag:Krizhevsky2013_nips_cnn], but we recommend the use of an independent test
set whenever feasible.

**Comment by agitter** [comment]
@brettbj what formatting problems were you having?  Was it git-related (e.g. merge conflicts) or markdown-related?

**Comment by brettbj** [comment]
Git-related, I have another open pull request with the same file which is causing issues:https://github.com/greenelab/deep-review/pull/577 I can either combine or wait for that to be decided on?

**Comment by cgreene** [comment]
@brettbj : Do you want to wrap this up now? All outstanding PRs against that text should be closed now.

**Comment by cgreene** [comment]
#768 would address this. Noting here to cross-link them.

**Comment by agitter** [comment]
Closed by #799
